src/app/tests/suites/TV_WakeOnLanCluster.yaml

```diff
@@ -32,4 +32,5 @@ tests:
       command: "readAttribute"
       attribute: "MACAddress"
       response:
-          value: "00:00:00:00:00"
+          constraints:
+              minLength: 3
```

Comment:
Why 3?  I would expect an octet string of exactly 6 octets here, so minLength and maxLength should both be 6, no?

Reply:
ok, you are right. I will fix in a followup since CI is taking forever and there is another PR dependent upon this one.

Reply:
https://github.com/project-chip/connectedhomeip/issues/25931